scripts/youtube/add_ads_video_assets.py

```
@@ -0,0 +1,342 @@
+                # Create asset (skip if video already exists as asset)
+                if video_id in existing_video_ids:
+                    print(f"   ⏭️  {locale} — asset already exists")
+                    # Still need to link to campaign
+                    # Find the existing asset resource name
+                    ga_service = client.get_service("GoogleAdsService")
+                    query = f"""
+                        SELECT asset.resource_name
+                        FROM asset
+                        WHERE asset.youtube_video_asset.youtube_video_id = '{video_id}'
+                        LIMIT 1
+                    """
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)